pkgs/misc/emulators/retroarch/default.nix

```diff
@@ -50,7 +50,7 @@ in stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  configureFlags = if stdenv.isLinux then [ "--enable-kms" ] else "";
+  configureFlags = stdenv.lib.optionals stdenv.isLinux [ "--enable-kms" "--enable-egl" ];
```

Comment:
Can you confirm that "egl" isn't detected by default? I thought in the past the configure script could guess it.

Reply:
I thought that it didn't pick it up when I tested it on aarch64. Maybe it does though. I'm still afraid to change anything until I understand why the check is failing, if it matters.